content/en/logs/explorer/calculate_queries.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Calculate Queries
+kind: documentation
+disable_toc: false
+private: true
+---
+
+## Overview
+
+Use calculated fields when you want to apply boolean logic, perform arithmetic operations, and/or manipulate strings while querying your logs.
```

Comment:
Calculated Fields allow you to calculate, compare, and transform your log data on-the-fly.